Client/Ecosia/Settings/EcosiaSettings.swift

```diff
@@ -210,3 +210,28 @@ final class HomepageSettings: Setting {
     }
 }
 
+// Ecosia: Unleash Shortcuts Experiment
```

Comment:
it's our own class. So technically the `// Ecosia:` would not be needed. Just FYI - Not a blocker at all.

Reply:
Added it to keep track of where the changes are mostly. Happy to remove it.